crates/holochain_zome_types/src/clone.rs

```diff
@@ -40,7 +40,7 @@ pub struct ClonedCell {
 
 /// Ways of specifying a clone cell in the context of an app.
 #[derive(Clone, Debug, Display, serde::Serialize, serde::Deserialize)]
-#[serde(untagged)]
```

Comment:
Was untagged before and accepted either DnaHash or RoleName, see [here](https://github.com/holochain/holochain-client-js/blob/b31403f3a274ddabc5750f9a487614b69fe7a6d6/src/api/admin/types.ts#L654).